contrib.json

```diff
@@ -54,19 +54,44 @@
         [ "npm install", "Download dependencies"]
       ]
     },
+    "upstream": {
```

Comment:
It feels off to have this as `contrib update upstream master` since we're not 'updating' upstream master, we're fetching/pulling/getting upstream master and updating local master vs remote master. Should we have `update local master` and `update remote master`?

It's also just two commands if we want to just not have a separate task for it.